extensions/wikia/UserLogin/UserLoginSpecialController.class.php

```diff
@@ -602,18 +612,32 @@ public function changePassword() {
 		$this->pageHeading = wfMessage('resetpass')->escaped();
 		$this->initializeTemplate();
 
-		$this->username = $this->request->getVal( 'username', '' );
-		$this->password = $this->request->getVal( 'password', '' );
-		$this->newpassword = $this->request->getVal( 'newpassword', '' );
-		$this->retype = $this->request->getVal( 'retype', '' );
-		$this->editToken = $this->request->getVal( 'editToken', '' );
-		$this->loginToken = $this->request->getVal( 'loginToken', '' );
-		$this->returnto = $this->request->getVal( 'returnto', '' );
+		$username = $this->request->getVal( 'username', '' );
+		$password = $this->request->getVal( 'password', '' );
+		$newPassword = $this->request->getVal( 'newpassword', '' );
+		$retype = $this->request->getVal( 'retype', '' );
+		$loginToken = $this->request->getVal( 'loginToken', '' );
+		$returnto = $this->request->getVal( 'returnto', '' );
+
+		// TODO: figure out why we were setting editToken twice, once via request var and once
```

Comment:
Setting it to match the core behavior didn't fix the issue in https://wikia-inc.atlassian.net/browse/SOC-351, but it didn't make it worse, so I'm going to wait till that ticket comes up in the backlog to troubleshoot this further.